### PR TITLE
[doc]rfct] example usage of list jobs

### PR DESCRIPTION
**Pull request by M3ssman**
Concerning issue #120

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -137,14 +137,15 @@ Listing all registered jobs
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Information, including the next scheduled execution, about the registered jobs of a deck-chores
-instance can be logged at once by sending ``SIGUSR1`` signal to the process, e.g. to one that runs
-in a container::
+instance can be logged by sending ``SIGUSR1`` signal to the *deck-chores*-process. For example,
+with a running *deck-chores* container named ``deck-chores_officer_1``, one needs to call::
```
Comment (funkyfuture) [inline docs/usage.rst:141]:
```suggestion
with a running *deck-chores* from a ``docker compose`` project, one needs to call::
```

File: docs/usage.rst [hunk 2]
```diff
@@ -137,14 +137,15 @@ Listing all registered jobs
...
-    docker kill --signal USR1 <CONTAINER>
+    docker kill --signal USR1 deck-chores_officer_1
 
-
-The output appears in ``deck-chores``' log target, that are the container's logs when it runs within
-one.
+In this case, output appears in ``deck-chores_officer_1`` log target, which are the container's logs
+accessible via ``docker logs deck-chores_officer_1``. The beginning of the output is marked by 
+``|SIGUSR1 received, echoing all jobs.``, followed by detailed information regarding every registered,
+active job.
```
Comment (funkyfuture) [inline docs/usage.rst:148]:
```suggestion
The output appears in ``deck-chores``' log target, that are the container's logs when it runs within
one.
```

here it gets just too verbose for my taste and the consistency of the deliberately compact docs.

**Review by funkyfuture — CHANGES_REQUESTED** [review]
thank you Uwe, i was quiet struggling with the verbosity of yor extension. do you think my proposals of change would still be helpful?

the docs aren't supposed to inform about processes, signals, containers, logs and so on; it assumes users are oriented i these regards and rather consult the resources that deal with the used technologies.

**Comment by M3ssman** [comment]
Yes of course I appreciate the feedback. 
For an experienced user it was way too verbose, but blame it on my educational point-of-view :slightly_smiling_face:

**Comment by funkyfuture** [comment]
a'ight, i'm bad at usign web based interfaces, hence i reworked the changes locally and merged all w/ 85a1da6ff7085ebbfc4108ed92bc9af879a8541b.

> but blame it on my educational point-of-view 🙂

i think it makes sense to have various types of user-facing documentation. as maintainer i only want to focus on the reference. i wouldn't say no to amend user stories / scenarios documentation chapters that take a narrative  approach.

**Pull request closed without merging**